### DISCUSSION TITLE
FusionCharge Integration

### DISCUSSION THREAD

**Discussion by ben-tsk** [wlcrs/huawei_solar#702] [Ideas]
Hi there,

im really searching for an integration for the Huawei FusionCharge. 

Is it possible to add a feature to this integration?

**Comment by Roving-Ronin** [comment]
@ben-tsk 

See: https://github.com/wlcrs/huawei_solar/issues/491

Doesn't help that hardly anyone has a SmartCharger at this point in time, let alone modbus register info is unconfirmed currently.
IF the modbus info could be sorted out, AND given the recent changes within HA in regards how sensors are polled that WLCRS/Thijs has then been able to add additional sensors (such as LUNA ESS battery temperature) in v1.4.0 (in testing) without affecting / slowing down the integration for everyone else... I would hope there would be the chance that Thijs will add support for SmartChargers into the integration (allowing things like stop / start charging and setting power limits etc).
